opencog/benchmark/AtomSpaceBenchmark.cc

```diff
@@ -1178,7 +1178,7 @@ timepair_t AtomSpaceBenchmark::bm_getOutgoingSet()
 #if HAVE_CYTHONX
```

Comment:
The reason this is HAVE_CYTHONX is because it doesn't support testing of loops for Python. The same comment is around all the python testing code for the loop test, for some reason.